apps/docs/components/ui/header/Header.tsx

```diff
@@ -20,7 +20,7 @@ const Header = () => {
         if (isMobileMenuOpen) {
             document.body.style.overflow = "hidden";
         } else {
-            document.body.style.overflow = "auto";
```

Comment:
visible is the default value, and adding auto here created issues when adding using usePreventScroll from react aria components